src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -53,6 +55,7 @@ export class AddScenarioModal implements OnInit, OnDestroy {
         private _settingsService: SettingsService, private _configService: ConfigService, private _toasterService: ToasterService,
         private _authService: AuthService) {
         this.newScenForm = _fb.group({
+            'regionid': new FormControl(null, Validators.required),
```

Comment:
You're actually setting this to the whole region, so the property should probably be named `region` instead